cookbook-react/recipes/tab-bars/tab-bar-1/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import './styles.scss';
+
+function TabBar() {
```

Comment:
Hmmm I'm not sure why you did it this way, but this TabBar is a hardcoded one :stuck_out_tongue: I would do something like this:
```ts
interface Option {
  label: string,
  id: string | number // Maybe you can use it 
}
// Btw, I'm not sure how to use dynamic keys in the interface.

interface Props {
  options: Option[] // I'm not sure if this is like this
}

function TabBar({ options } : Props ) {
  const [selection, setSelection] = useState(null);

  return (
    <div id="tab-bar-1" className="tab-bar-container">
      <div className="tab-bar-wrapper">
        <span className={`selector selected-${selection}`} />
        <div className="tabs">
          {options.map((option, idx) => {
            <button
              className={cn("option", { "selected": selection == idx })}
              onClick={() => setSelection(idx)} // Maybe you can use the id instead of the index.
            >
              {option.label}
            </button>
          })}
        </div>
      </div>
    </div>
  )
}

TabBar.defaultProps: {
  options: [{ id: 1, label: 'Sección 1', { id: 2, label: 'Sección 2' }, { id: 3, label: 'Sección 3' }}]
}
```